odbc_fdw.c

```diff
@@ -155,9 +182,208 @@ odbc_fdw_handler(PG_FUNCTION_ARGS)
 	fdwroutine->IterateForeignScan = odbcIterateForeignScan;
 	fdwroutine->ReScanForeignScan = odbcReScanForeignScan;
 	fdwroutine->EndForeignScan = odbcEndForeignScan;
+	fdwroutine->ImportForeignSchema = odbcImportForeignSchema;
 	PG_RETURN_POINTER(fdwroutine);
 }
 
+void
+init_odbcFdwOptions(odbcFdwOptions* options)
+{
+	options->dsn          = NULL;
+	options->driver       = NULL;
+	options->host         = NULL;
+	options->port         = NULL;
+	options->database     = NULL;
+	options->username     = NULL;
+	options->password     = NULL;
+	options->schema       = NULL;
+	options->table        = NULL;
+	options->prefix       = NULL;
+	options->sql_query    = NULL;
+	options->sql_count    = NULL;
+	options->mapping_list = NIL;
```

Comment:
No big deal but it is pretty standard to just use `memset(options, 0, sizeof odbcFdwOptions)`. Actually recommendable.